ludwig/benchmarking/summarize.py

```diff
@@ -32,12 +36,73 @@ def summarize_metrics(
             metric_diff = build_metrics_diff(dataset_name, base_experiment, experimental_experiment, local_dir)
             metric_diffs.append(metric_diff)
 
-            base_path = os.path.join(local_dir, dataset_name, base_experiment, "resource_usage_metrics")
-            experimental_path = os.path.join(local_dir, dataset_name, experimental_experiment, "resource_usage_metrics")
+            base_path = os.path.join(local_dir, dataset_name, base_experiment)
+            experimental_path = os.path.join(local_dir, dataset_name, experimental_experiment)
             resource_usage_diff = build_resource_usage_diff(
                 base_path, experimental_path, base_experiment, experimental_experiment
             )
             resource_usage_diffs.append(resource_usage_diff)
         except Exception:
             logging.exception(f"Exception encountered while creating diff summary for {dataset_name}.")
-    return metric_diffs, resource_usage_diffs
+    shutil.rmtree(local_dir, ignore_errors=True)
+    export_and_print(dataset_list, metric_diffs, resource_usage_diffs)
+    return dataset_list, metric_diffs, resource_usage_diffs
+
+
+def export_and_print(
```

Comment:
nit: Please add a docstring that describes what this function does -- it looks like there's a lot of interesting things happening with files getting saved and things getting logged.